alibi_detect/saving/schemas.py

```diff
@@ -19,18 +19,10 @@
 from pydantic import BaseModel, validator
 
 from alibi_detect.utils.frameworks import Framework
-from alibi_detect.utils._types import (Literal, NDArray, supported_models_all, supported_models_tf,
+from alibi_detect.utils._types import (Literal, supported_models_all, supported_models_tf,
                                        supported_models_sklearn, supported_models_torch, supported_optimizers_tf,
                                        supported_optimizers_torch, supported_optimizers_all)
-
-
-# Custom validators (defined here for reuse in multiple pydantic models)
-def coerce_int2list(value: int) -> List[int]:
```

Comment:
Moved to a new `validators.py` file, since we are beginning to add lots of these...